src/main/java/by/itechart/api/entity/User.java

```diff
@@ -13,7 +12,6 @@
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-@ApiModel(value = "user model")
```

Comment:
Since on controller level we are using DTO classes we should place Swagger annotations in the DTO classes